lib/catalog/order_item_sanitized_parameters.rb

```diff
@@ -73,7 +59,7 @@ def mask_value?(field)
     end
 
     def fields
-      @fields ||= service_plan_schema.with_indifferent_access.dig(:schema, :fields)
+      @fields ||= ServicePlanFields.new(order_item).process.fields
```

Comment:
@bzwei How do we account for job templates that might not have a Survey Spec will this call return an empty array or fail?

Reply:
it returns an empty array